### DISCUSSION TITLE
Error pada saat running shiny

### DISCUSSION THREAD

**Discussion by fxpeterbs31** [teamalgoritma/community#248] [Q&A] [answered]
Halo, saya mau tanya. 

Pada saat saya ingin mempublish shiny, deployment sudah selesai dan berhasil. Akan tetapi, tidak bisa dibuka di web dan menghasilkan error seperti ini.
<img width="328" alt="2022-08-04 16_44_35-Error" src="https://user-images.githubusercontent.com/107398327/182817378-bd609375-8bb8-46ff-b1ed-2e1c7186aaf3.png">
Kira2 masalahnya terletak dimana ya dan untuk membenarkannya bagaimana ya?  

Terima kasih.

**Comment by triarm** [comment, marked as answer]
Hi @fxpeterbs31 ,

Apakah saat menjalankan dashboard secara lokal mengalami error yang sama? atau hal ini hanya terjadi saat melakukan deployment di shinyapps?

**Reply by fxpeterbs31** [reply]
Tidak ada mas, hanya di shinyapps a
saja.

**Reply by triarm** [reply]
Apakah bisa dilampirkan pesan error yang muncul saat deployment di shinyapps pada bagian console?
